assets/js/gamesData.json

```diff
@@ -2102,22 +2102,22 @@
  "gameTitle": "Guess_The_Song",
   "gameUrl": "Guess_The_Song",
   "thumbnailUrl": "Guess_The_Song.png"
-  },"421":{"gameTitle": "Pen_Pointer_Fight",
-  "gameUrl": "PenPointerFight",
-  "thumbnailUrl": "PenPointerFight.png"
-},"422":{
-  "gameTitle": "MathQuiz",
-  "gameUrl": "MathQuiz",
-  "thumbnailUrl": "MathQuiz.png"
-},"423":{
-   "gameTitle": "Soccer",
-   "gameUrl": "Soccer",
-   "thumbnailUrl": "Soccer"
-}
+  }
 },"417":{
   "gameTitle": "Brick Buster",
    "gameUrl": "Brick Buster",
    "thumbnailUrl": "Brick.png"
- }
+ },"418":{
+  "gameTitle": "Soccer",
+  "gameUrl": "Soccer",
+  "thumbnailUrl": "Soccer"
+},"419":{"gameTitle": "Pen_Pointer_Fight",
+"gameUrl": "PenPointerFight",
+"thumbnailUrl": "PenPointerFight.png"
+},"420":{
```

Comment:
place this one in above space for 408 and remove from here also change label to 408